### ISSUE TITLE
Calibration of Multiple Cameras with Overlapping

### ISSUE THREAD

**Issue by Nick-0814** [ethz-asl/kalibr#685]
The device I am using is a dual-fisheye camera with overlapping front and rear parts, and I am trying to calibrate the external parameters between the two cameras 
I made sure that the rosbag I recorded had obvious overlaps, but during running kalibr, I was prompted to have a bug (The cameras are not connected through mutual target observations! Please provide another dataset)

Terminal command:rosrun kalibr kalibr_calibrate_cameras --target sample.yaml --models omni-radtan omni-radtan --topics /cam0/image_raw /cam1/image_raw --bag try.bag --bag-freq 7.5

![example](https://github.com/ethz-asl/kalibr/assets/111438180/382223c0-8d85-428b-bc96-428b73c6b6bc)
![problem](https://github.com/ethz-asl/kalibr/assets/111438180/35cd9955-f1e6-495e-a6c3-8358ea05d9aa)

**Comment by skohlbr** [comment]
See my comment here: https://github.com/ethz-asl/kalibr/issues/231#issuecomment-2095568096

tl;dr: Made it work by adding more cameras so there can be mutual observations.

**Comment by goldbattle** [comment]
Adding an additional temporary camera is a way to solve this limited overlap. At the edges the detection and use of targets becomes very tough due to the high distortion so it is also better to add an intermediary camera in your case to help improve this also.
